doc/source/user_guide/meshing/new_meshing_workflows.rst

```diff
@@ -699,3 +699,60 @@ Duplicate tasks
     watertight.import_geometry.insertable_tasks.import_boi_geometry.insert()
     assert watertight.import_boi_geometry.arguments()
     assert watertight.import_boi_geometry_1.arguments()
+
+
+Current meshing workflow
+------------------------
+Use the ``current_workflow`` method to access an already loaded workflow.
+The following example shows how to use this method.
+
+Current workflow
+~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    meshing.current_workflow
+
+.. Note::
+   ``current_workflow`` raises attribute error when no workflow is initialized.
```

Comment:
```suggestion
   The ``current_workflow`` method raises an attribute error when no workflow is initialized.
```

Reply:
It should have been attribute, thank you :)